package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "web3modal",
-  "version": "4.0.0",
+  "version": "5.1.0",
```

Comment:
just double checking: is this correct?

Reply:
yes, we should enforce this with a danger rule though.